rotkehlchen/chain/evm/decoding/efp/decoder.py

```diff
@@ -0,0 +1,158 @@
+
+logger = logging.getLogger(__name__)
+log = RotkehlchenLogsAdapter(logger)
+
+LIST_OP_TOPIC: Final = b"\xb0gc\x0e9\x08\xf1f\x06\x9cwvbP\xc5SAN'E\xe7B\xa5b\xd53\xf2\x97D\xa9\x1as"  # noqa: E501
+
+
+class EfpCommonDecoder(DecoderInterface, ABC):
+
+    def __init__(
+            self,
+            evm_inquirer: 'EvmNodeInquirer',
+            base_tools: 'BaseDecoderTools',
+            msg_aggregator: 'MessagesAggregator',
+            list_records_contract: 'ChecksumEvmAddress',
+    ) -> None:
+        super().__init__(
+            evm_inquirer=evm_inquirer,
+            base_tools=base_tools,
+            msg_aggregator=msg_aggregator,
+        )
+        self.list_records_contract = list_records_contract
+
+    def _get_address_for_slot(self, slot: int) -> 'ChecksumEvmAddress | None':
+        """Get address associated with the specified slot.
+        Returns an address or None if unable to match an address to this slot."""
+        with GlobalDBHandler().conn.read_ctx() as cursor:
+            if (address := globaldb_get_unique_cache_value(
+                    cursor=cursor,
+                    key_parts=(CacheType.EFP_SLOT_ADDRESS, str(slot)),
+            )) is not None:
+                return string_to_evm_address(address)
+
+        try:
+            if (address := to_checksum_address(self.evm_inquirer.contracts.contract(self.list_records_contract).call(  # noqa: E501
+                node_inquirer=self.evm_inquirer,
+                method_name='getListUser',
+                arguments=[slot],
+            ))) == ZERO_ADDRESS:
+                log.error(f'Failed to retrieve address for EFP slot {slot} on {self.evm_inquirer.chain_id.label()}')  # noqa: E501
+                return None
+        except RemoteError as e:
+            log.error(f'Failed to retrieve address for EFP slot {slot} on {self.evm_inquirer.chain_id.label()} due to {e!s}')  # noqa: E501
+            return None
+
+        with GlobalDBHandler().conn.write_ctx() as write_cursor:
+            globaldb_set_unique_cache_value(
+                write_cursor=write_cursor,
+                key_parts=(CacheType.EFP_SLOT_ADDRESS, str(slot)),
+                value=address,
+            )
+        return address
+
+    def _decode_list_op_events(self, context: DecoderContext) -> DecodingOutput:
+        """Decode EFP list operation events."""
+        if context.tx_log.topics[0] != LIST_OP_TOPIC:
+            return DEFAULT_DECODING_OUTPUT
+
+        if (address := self._get_address_for_slot(
+                slot=(slot := int.from_bytes(context.tx_log.topics[1])),
+        )) is None:
+            log.error(
+                f'Unable to decode EFP list op event due to unknown slot {slot} '
+                f'in transaction {context.transaction}',
+            )
+            return DEFAULT_DECODING_OUTPUT
+
+        if not self.base.is_tracked(address):
+            log.debug(
+                f'Skipping EFP list op event for untracked address {address} '
```

Comment:
chain id log here too.

Reply:
this error also uses `context.transaction`